src-built-in/components/adhocComponentForm/app.jsx

```diff
@@ -95,10 +96,8 @@ class AdHocComponentForm extends React.Component {
 	 * @memberof AdHoc
 	 */
 	cancel() {
-		this.setState({
```

Comment:
This was deleted because the dialog was keeping the information in the input boxes preserved when the user pressed "Cancel". Expected behavior for a Cancel would be to clear the fields, that way a user doesn't need to go into the fields again and delete the text.